### DISCUSSION TITLE
Should we add a VO column in the Jobs table ?

### DISCUSSION THREAD

**Discussion by simon-mazenoux** [DIRACGrid/DIRAC#7197] [Q&A] [answered]
With diracx, groups will be unique only in regard to their VO. Therefore, it seems like a good idea to add the VO field in the Jobs table. Also, I know that the VO can be retrieved for any job from the JDL, but I don't think this is particularly handy.

With this new column, a user could now get the VO associated with jobs by calling the getJobsAttributes method of the JobMonitoringHandler instead of having to parse the JDL to get this field. 

One other benefit from this change would be to be able to make SQL queries that could `GROUP BY VO`.

What do you think ?

**Comment by chaen** [comment]
I think it's a good idea

**Reply by fstagni** [reply, marked as answer]
Done.
